Project.toml

```diff
@@ -37,6 +36,7 @@ Reexport = "0.1, 0.2"
 SortingAlgorithms = "0.1, 0.2, 0.3"
 TableTraits = "0.4, 1"
 Tables = "1.2"
+julia = "1"
```

Comment:
why put it last?

Reply:
I did not change this intentionally :-/. Not sure how that happened, but I had to `] instantiate` and did it on both julia 1.5 and 1.6, maybe that did something? I'll change it back